src/components/Button.tsx

```diff
@@ -23,6 +23,7 @@ export interface ButtonProps extends BeamButtonProps, BeamFocusableProps {
   /** Denotes if this button is used to download a resource. Uses the anchor tag with the `download` attribute */
   download?: boolean;
   contrast?: boolean;
+  xss?: Xss<"borderColor">;
```

Comment:
**fyi:** I'm hoping we can remove the XSS property here, but just for an fyi... The XSS property is intended to limit the style properties that can be put on an element. As it is right now, we are allowing the user to put any styles on the element via the `xss` property, which we don't really want to do. To make this actually type safe, then we'd need to update the Button function component.
It would look something more like:
```
import { Only, Xss } from "src/Css";

// Create a type to make the same XSS easier to reference
type ButtonXss = Xss<"borderColor">;

// Use a generic in the props for the XSS type.
interface ButtonProps<X> { xss?: X; }

// Have the generic extend our `Only` TS helper which limits the values that can be passed into the XSS based on what we defined.
export function Button<X extends Only<ButtonXss, X>>(props: ButtonProps<X>) {
```